libs/designer/src/lib/ui/panel/recommendation/recommendationPanelContext.tsx

```diff
@@ -57,14 +57,17 @@ export const RecommendationPanelContext = (props: RecommendationPanelContextProp
   const { data: allConnectors } = useAllConnectors();
   const selectedConnector = allConnectors?.find((c) => c.id === selectedOperationGroupId);
 
+  // hide actions type filter if we don't have any operations for the browse view
+  const hideActionTypeFilter = (!allOperations || allOperations.length === 0) && !searchTerm;
```

Comment:
With the current setup it looks like you're going to be seeing the action type filter pop in and out or rendering when you have things in the search bar and not, rather than just disabling the field.
I'm fine with either, logic apps will not be using this scenario, just wanted to bring it up.

Reply:
Yeah I believe we're okay with that given the change in the list nature (one list shows connectors, the other shows operations)